MatrixKit/Categories/NSBundle+MatrixKit.m

```diff
@@ -25,6 +25,11 @@ @implementation NSBundle (MatrixKit)
 + (NSBundle*)mxk_assetsBundle
 {
     NSBundle *bundle = [NSBundle bundleForClass:[MXKViewController class]];
+    if ([[bundle.bundleURL pathExtension] isEqualToString:@"appex"]) {
```

Comment:
Can you create a method in NSBundle+MatrixKit.h like ```mxk_bundleForClass:``` that will do this trick?
We will probably need it in more and more places.